tests/pipelines/audioldm2/test_audioldm2.py

```diff
@@ -469,29 +469,6 @@ def test_save_load_optional_components(self):
         # increase tolerance from 1e-4 -> 2e-4 to account for large composite model
         super().test_save_load_optional_components(expected_max_difference=2e-4)
 
-    def test_to_dtype(self):
```

Comment:
Once https://github.com/huggingface/transformers/pull/25682 is merged, overriding this test won't be necessary